<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #EEEFE0;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="All_task">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1011</width>
      <height>591</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Mj_Bomba</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget{
    background-color: #D1D8BE; 
    border: 1px solid #D3D3D3;
    border-radius:10px;
}

QTabBar::tab {
    background-color: #C1D9BB;
    color: black;
    padding: 5px;
    margin-right: 2px;
    border: 1px solid #B0B0B0;
    border-top-left-radius: 10px;
    border-top-right-radius:10px;
}

QTabBar::tab:selected {
    background-color: #A9CBA3; 
    border-bottom: 1px solid #A9CBA3;
}

QPushButton {
    background-color: #C1D9BB;
    border: 1px solid #B0B0B0;
    border-radius: 5px;
    padding: 5px;
}

QPushButton::hover {
    background-color: #A9CBA3; 
}
QTabBar::tab {
    min-width: 100px; /* حداقل عرض تب */
    min-height: 30px; /* حداقل ارتفاع تب */
    max-width: 150px; /* حداکثر عرض تب */
    max-height: 40px; /* حداکثر ارتفاع تب */
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="All_taskPage1_2" native="true">
     <attribute name="title">
      <string>All</string>
     </attribute>
     <widget class="QFrame" name="all_frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1001</width>
        <height>541</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: #D1D8BE;
	border-radius: 20px solid;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	color: #262626;
	background-color: #D1D8BE;
	border : 0.1;
	border-radius : 15px ;
}</string>
       </property>
       <property name="text">
        <string>Add Task</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>C:/Users/mahrayan/Downloads/icons8-plus-50 (1).png</normaloff>C:/Users/mahrayan/Downloads/icons8-plus-50 (1).png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="All_taskPage2_2" native="true">
     <attribute name="title">
      <string>Today</string>
     </attribute>
     <widget class="QFrame" name="today_frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1001</width>
        <height>541</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: #D1D8BE;
	border-radius: 20px solid;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	color: #262626;
	background-color: #D1D8BE;
	border : 0.1;
	border-radius : 15px ;
}</string>
       </property>
       <property name="text">
        <string>Add Task</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>C:/Users/mahrayan/Downloads/icons8-plus-50 (1).png</normaloff>C:/Users/mahrayan/Downloads/icons8-plus-50 (1).png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="All_taskPage3_2" native="true">
     <attribute name="title">
      <string>Complete</string>
     </attribute>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1001</width>
        <height>541</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: #D1D8BE;
	border-radius: 20px solid;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
